calib_sequence: '0012345'
velocity_control_enabled: true